data/presets_default.json

```diff
@@ -417,53 +417,69 @@
+    "bridge":                                  "stones",
+    "bridge_stones":                           3,
+    "triforce_hunt":                           false,
+    "logic_rules":                             "glitchless",
+    "reachable_locations":                     "all",
+    "bombchus_in_logic":                       false,
+    "one_item_per_dungeon":                    false,
+    "trials_random":                           false,
+    "trials":                                  0,
+    "skip_child_zelda":                        true,
+    "no_escape_sequence":                      true,
+    "no_guard_stealth":                        true,
+    "no_epona_race":                           true,
+    "skip_some_minigame_phases":               true,
+    "useful_cutscenes":                        false,
+    "complete_mask_quest":                     false,
+    "fast_chests":                             true,
     "logic_no_night_tokens_without_suns_song": false,
+    "free_scarecrow":                          true,
+    "fast_bunny_hood":                         true,
+    "start_with_rupees":                       false,
+    "start_with_consumables":                  true,
+    "starting_hearts":                         5,
+    "chicken_count_random":                    false,
+    "chicken_count":                           5,
+    "big_poe_count_random":                    false,
+    "big_poe_count":                           1,
+    "shuffle_kokiri_sword":                    true,
+    "shuffle_ocarinas":                        false,
+    "shuffle_gerudo_card":                     false,
+    "shuffle_song_items":                      "song",
+    "shuffle_cows":                            false,
+    "shuffle_beans":                           false,
+    "shuffle_medigoron_carpet_salesman":       false,
+    "shuffle_interior_entrances":              "off",
+    "shuffle_grotto_entrances":                false,
+    "shuffle_dungeon_entrances":               false,
+    "shuffle_overworld_entrances":             false,
+    "owl_drops":                               false,
+    "warp_songs":                              false,
+    "spawn_positions":                         true,
+    "shuffle_scrubs":                          "off",
+    "shopsanity":                              "off",
+    "tokensanity":                             "off",
+    "shuffle_mapcompass":                      "startwith",
+    "shuffle_smallkeys":                       "dungeon",
+    "shuffle_hideoutkeys":                     "vanilla",
+    "shuffle_bosskeys":                        "dungeon",
+    "shuffle_ganon_bosskey":                   "on_lacs",
+    "lacs_condition":                          "vanilla",
+    "enhance_map_compass":                     false,
+    "mq_dungeons_random":                      false,
+    "mq_dungeons":                             0,
     "disabled_locations": [
```

Comment:
For consistent style, this bracket should be indented further.

Reply:
Hmm, I thought it should be, but when I committed the first time, it showed that the indent was originally where i changed it back to

Reply:
It was, but you changed this preset to column indentation.